direct/io-cassandra/pom.xml

```diff
@@ -119,10 +119,16 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>cz.o2.proxima</groupId>
+      <artifactId>proxima-direct-io-serialization</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
     <dependency>
       <groupId>com.datastax.cassandra</groupId>
       <artifactId>cassandra-driver-core</artifactId>
-      <version>3.8.0</version>
+      <version>3.11.0</version>
```

Comment:
NIT: we should probably bump this separately just to have clean history

Reply:
moved